peas/weather_ext.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3

```

Comment:
I don't understand this file name. Is 'ext' for 'external'?

Reply:
Yeah, I can change it to be weather_external.py if that would be better.